Microsoft.HealthVault.Client.nuspec

```diff
@@ -17,33 +17,32 @@
     <dependencies>
       <group targetFramework="netstandard1.4">
       	<dependency id="Microsoft.HealthVault" version="$version$" exclude="Build,Analyzers" />
```

Comment:
>$version$ [](start = 55, length = 9)

Microsoft.HealthVault dependencies should probably use [$version$] to target the exact version.